pkg/compact/downsample/downsample.go

```diff
@@ -431,15 +471,18 @@ func downsampleBatch(data []sample, resolution int64, add func(int64, *aggregato
 }
 
 // downsampleAggr downsamples a sequence of aggregation chunks to the given resolution.
-func downsampleAggr(chks []AggrChunk, buf *[]sample, mint, maxt, resolution int64) ([]chunks.Meta, error) {
+func downsampleAggr(chks []AggrChunk, buf *[]sample, mint, maxt, inResolution, outResolution int64) ([]chunks.Meta, error) {
```

Comment:
in consistent naming. In `targetChunkCount` you have `inRes` and `outRes`. Please choose one style. OK with both.